resteasy-jaxrs-testsuite/src/test/java/org/jboss/resteasy/test/nextgen/interceptors/GzipTest.java

```diff
@@ -175,9 +181,14 @@ public void testRawStreams() throws Exception
    @Test
    public void testProxy() throws Exception
    {
-      ResteasyClient client = new ResteasyClientBuilder().build();
+      ResteasyClient client = getClient();
       ResteasyWebTarget target = client.target(generateBaseUrl());
       IGZIP proxy = target.proxy(IGZIP.class);
+      // register gzip explicitly
+      ResteasyProviderFactory providerFactory = ResteasyProviderFactory.getInstance();
+      providerFactory.registerProvider(AcceptEncodingGZIPInterceptor.class);
+      providerFactory.registerProvider(GZIPDecodingInterceptor.class);
+      providerFactory.registerProvider(GZIPEncodingInterceptor.class);

```

Comment:
I think the lines above are not needed now that you use the getClient() method above

Reply:
Fixed, I missed it in vim. Thank you!